packages/libs/avector/README.md

```diff
@@ -0,0 +1,49 @@
+[announcement post]: https://hash.dev/blog/announcing-error-stack
+[crates.io]: https://crates.io/crates/error-stack
+[libs.rs]: https://lib.rs/crates/error-stack
+[rust-version]: https://www.rust-lang.org
+[documentation]: https://docs.rs/error-stack
+[license]: https://github.com/hashintel/hash/blob/main/packages/libs/error-stack/LICENSE.md
```

Comment:
Non of these links are correct (besides `rust-lang.org`)

Reply:
fixed in [c4df4db](https://github.com/hashintel/hash/pull/1399/commits/c4df4db830fe09e58950b57e168c1f80e79fc374)

Reply:
The links are still not correct as the crate is not published (yet)

Reply:
Yes, my thinking was to reference them already in the documentation, and once published, they will point to the crate, ofc I can remove them. (same with the crates.io and lib.rs badge)

Reply:
Yes, let's remove them from the readme. We should avoid dead links in a readme.

Reply:
removed in [07829cc](https://github.com/hashintel/hash/pull/1399/commits/07829cc38d3009ccd4bd8329f63976ea7e4e1e66)